README.md

```diff
@@ -62,6 +62,37 @@ _For both Windows and Mac : To verify that installation was successful, `protoc
 
 ### Config Format
 
+#### Exclude Paths: ####
+
+This allows the user to blacklist a certain set of paths for a given path. As a result those paths would be eliminated while grabbing content, there are certain rules that need to be followed,
+
+1. Exclude Paths only accepts relative paths and not absolute paths.
+
+    For example,
+
+    Let's say given path : `"/content/someContent"`  and you would like to exclude `"/content/someContent/someOtherContent/pdfs"`
+
+    In this case, providing exclude paths as
+    `"/content/someContent/someOtherContent/pdfs"` or `"/someOtherContent/pdfs"` is INVALID.
+
+    Correct way is `"someOtherContent/pdfs"`. So your path configuration would be as follows:
+    ```
+    {
+        "path" : "/content/someContent",
+        "excludePaths" :
+        [
+            "someOtherContent/pdfs"
+        ]
+    }
+    ```
+    You could also provide multiple exclusions for a single path(similar to workflow config ids)
+
+2. Following are INVALID characters to be provided as part of Exclusion Paths,
+
+    relative paths provided cannot begin or end with `/, ./ or \`
+
+3. `Exclude Paths is not a mandatory field` so it can be omitted from path configuration as well.
+
```

Comment:
Better:

### Config Format

A `json` object is used by the PUT to the client to tell it what to request from the server. An example configuration:

```json
{
    "serverUsername" : "<username>",
    "serverPassword" : "<password>",
    "serverHost" : "some.other.server",
    "serverPort" : "4502",
    "pathConfigurations" :  [
        {
            "path" : "/content/someContent",
        },
        {
            "path" : "/content/someContent",
            "excludePaths" :
            [
                "someOtherContent/someExcludeContent"
            ]
        },
        {
            "path" : "/content/dam/someDamContent",
            "excludePaths":
                [
                    "someContent/someExcludeContent",
                    "someContent/someOtherExcludeContent"
                ],
            "workflowConfigIds" :
                [
                    "/etc/workflow/launcher/config/update_asset_mod",
                    "/etc/workflow/launcher/config/update_asset_create",
                    "/etc/workflow/launcher/config/dam_xmp_nested_writeback",
                    "/etc/workflow/launcher/config/dam_xmp_writeback"
                ]
        }
    ]
}
```


#### Required fields

* **serverHost**: The server that the client should get its content from.
* **serverPort**: The port to connect to on the server that the client should use.
* **serverUsername**: The username the client should use to authenticate against the server.
* **serverPassword**: The password the client should use to authenticate against the server.
* **pathConfigurations**: The list of paths and their options to pull from the server.
  * **path**: The path to recursively grab content from.


#### Optional fields

Under "path configurations"

* **excludePaths**: This allows excluding specific subpaths from what will be retrieved from the parent path. See more detail below.
* **workflowConfigIds**: Before the client retrieves content for the path from the server, it will make sure that the specified workflows are disabled. They will be re-enabled when all content specifying that workflow has finished copying. (Grabbit handles the situation of multiple paths specifying "overlapping" workflows.) This is particularly useful for areas like the DAM where a number of relatively expensive workflows will just "redo" what is already being copied.


#### Exclude Paths

Exclude Paths allow the user to exclude a certain set of subpaths for a given path while grabbing content. They can only be **relative** to the "path".

For example, let's say you have
```json
{ "path" : "/content/someContent" }
```
and you would like to exclude `/content/someContent/someOtherContent/pdfs`

Valid:
```json
{
    "path" : "/content/someContent",
    "excludePaths" :
    [
        "someOtherContent/pdfs"
    ]
}
```
Invalid:
```json
{
    "path" : "/content/someContent",
    "excludePaths" :
    [
        "/content/someContent/someOtherContent/pdfs",
        "/someOtherContent/pdfs"
        "./someOtherContent/pdfs"
    ]
}
```